### PR TITLE
Add simple check for non-null and non-empty lobby host value.

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Functional Changes

[ ] New map or map update
[ ] New Feature
[ ] Feature update or enhancement
[ ] Feature Removal
[x] Code Cleanup or refactor
[ ] Configuration Change
[ ] Bug fix:  
[ ] Other:

### CODE REVIEW

**Comment by prastle** [comment]
In layman's terms will this fix the crashed ghosted bots?

**Comment by DanVanAtta** [comment]
No, ghosting is a much deeper problem rooted somewhere in some gnarly/complex Java socket code that was written 17 years ago :grin: 

This check is validity constraint so that the code blows up sooner rather than later; we get a more sensible error message sooner in the system rather than later with something else failing.

**Comment by prastle** [comment]
Sigh KK

**Comment by DanVanAtta** [comment]
@prastle this PR should though, at least, once the backend infrastructure is used: https://github.com/triplea-game/triplea/pull/5252

Of particular note, the '[GameReaper](https://github.com/triplea-game/triplea/pull/5252/files#diff-60b7dd1a97e376266cb0d5c32a9b5ef9)' that collects dead games and removes them.

It'll be a pretty big lift though to go from the using the classic-java lobby of today to use the http server with game reaper, but a big step is taken in that direction.

The fix for ghosting is going to be two fold:
(1) better handling of lost connections
(2) games will be required to send a 'keep-alive' message, if one is not received then game-reaper should kick in and remove them.

With luck too this should allow for an auto-reconnect to lobby.
Game hosts will send keep-alives, if the lobby dies, game hosts will see no response. When lobby is back, the keep alive message will be acknowledged with a 'false' instead of true. At that point the game hosts will re-post their games to the resurrected lobby.
